src/utils/network-tests.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Validates if we're running end-2-end (network) tests
+ */
+export default function runningNetworkTests(): boolean {
+  if (typeof process !== 'undefined') {
+    // process variable exists => NodeJS
+    // eslint-disable-next-line no-process-env
+    return process.env.E2ETESTS !== 'undefined' && process.env.E2ETESTS === '1';
+  }
+  if (typeof globalThis !== 'undefined') {
+    // not in node, therefore CF worker or miniflare
+    return ( (globalThis as any).MINIFLARE
```

Comment:
Yeah, I agree. I think we’d rather have another ENV variable that needs to be set than changing the behavior just because the developer is using a specific runtime and than gets divergent behavior.